HISTORY.rst

```diff
@@ -0,0 +1,23 @@
+
+.. :changelog:
+
+2020.1.0 (2020-06-23)
++++++++++++++++++++++
+
+- Add new available resen-core: 2020.1.0
+- bug fixed when importing a bucket: now tar files of mounted paths are removed after a successful import.
+- bug fixed when giving sudo privileges to locally run resen.
```

Comment:
```suggestion
- fixed bug causing enabling sudo access for user jovyan failed
```

Reply:
now it looks better. BTW, there are other changes from 2019.1.0 to 2019.1.1 that I didn't include.